controllers/cohortdata.go

```diff
@@ -17,8 +18,9 @@ type CohortDataController struct {
 	cohortDataModel models.CohortDataI
 }
 
-type PrefixedConceptIds struct {
+type RequestBody struct {
```

Comment:
please adjust to use the more extensive structure defined in the ticket (with `variables`, etc).

```
curl -d '{"variables": [
             {variable_type: "concept", prefixed_concept_id: "ID_2000000324"},
             {variable_type: "concept", prefixed_concept_id: "ID_2000006885"},
             {variable_type: "custom_dichotomous", cohort_ids: [cohortX_id, cohortY_id]},
             {variable_type: "custom_dichotomous", cohort_ids: [cohortM_id, cohortN_id]},
            ...
           ]
         }'
```

Reply:
done